### DISCUSSION TITLE
Could we disable LP solver inside SAT?

### DISCUSSION THREAD

**Discussion by C8504** [google/or-tools#2666] [CP-SAT questions] [answered]
Hi folks,

When solving a big binary or integer problem by SAT solver, could we disable LP?

thanks.

John

**Comment by ghost** [comment, marked as answer]
Maybe `solver.parameters.linearization_level = 0`

**Reply by lperron** [reply]
exactly. Note that this only work in single thread. I still recommend solving with 8 or 16 workers.

**Reply by C8504** [reply]
thanks. single worker with linearization_level:0 will disable LP.
then how to disable LP when solving with 8-16 workers? Thanks again.

**Reply by lperron** [reply]
You cannot. We control the portfolio of configurations.

**Reply by C8504** [reply]
Thanks, then linearization_level = 1 or 2 when using mutiple workers still works to control the size of LP as the document?

**Comment by lperron** [comment]
yes
Laurent Perron | Operations Research | ***@***.*** | (33) 1 42 68 53
00
